README.md

```diff
@@ -31,13 +31,26 @@ file [here](examples/unit_test_example.sh).
 The collection of helper libraries are expected to be loaded using the
 provided [src/bootstrap.sh](bootstrap) script.
 
-Currently, there are 2 supported modules. To read more about what each of the modules provide,
+The bootstrap's script responsibilities are to get a local copy of this repository, initialize
+the `loader` module, and load
+
+The `loader` module is a special one, and it's main function is to serve as an entrypoint to the
+library.
+It provides the `cicd::loader::load_module` functions, that must be used to load the different
+modules.
+
+This is all handed by the boostrap script
+
+To read more about what each of the modules provide,
 please read through the documents linked through the library IDs in the following table:
 
-| Library ID                                            | Description                                                                |
-|-------------------------------------------------------|----------------------------------------------------------------------------|
-| [container](docs/cicd_tools/container_lib.md)         | Provides wrapper functions for invoking container engine agnostic commands |
-| [image_builder](docs/cicd_tools/image_builder_lib.md) | Provides helper functions to simplify the image building process           |
+| Library ID                          | Description                                                                |
```

Comment:
Each of the "Description" lines starts with either "Provides wrapper functions" or "Provides helper functions" so you could slim that down by changing the column name to "Provided Functionality" and then the individual lines could be:

- Container engine agnostic commands
- Simplify the image building process

Also you've got duplicates in there I think might be a mistake? Like log, loader, and container all have the same description, as do image builder and common.

Reply:
Yes to all, I've simplified the redundant description and fixed the dups, they were leftovers indeed.